### PR TITLE
Update environment.yml file to current HUB package versions

### PR DESCRIPTION
**Pull request by elehmer**
This PR updates the `environment.yml` file used by the testing environment to the package versions currently on the CA:AE HUB server. Also removes `black` and `flake8` as they are not used at this time.

Also updates `requirements.txt` to match `environments.yml`

### CODE REVIEW

**Comment by elehmer** [comment]
@bkg Let me know if we should put all of the `environment.yml` packages in `requirements.txt`.

**Comment by bkg** [comment]
@elehmer thanks for sorting this out. Yes, it would be good if those two package lists are in sync.

**Comment by elehmer** [comment]
> @elehmer thanks for sorting this out. Yes, it would be good if those two package lists are in sync.

Done...

**Comment by elehmer** [comment]
@bkg I think I have this all set right but please let me know if not.

File: setup.cfg [hunk 1]
```diff
@@ -21,16 +21,20 @@ install_requires =
     geoviews
     holoviews
     hvplot
+    intake
     intake-xarray
+    jinja2
```
Comment (bkg) [inline setup.cfg:26]: I don't believe this is used anywhere.

File: setup.cfg [hunk 2]
```diff
@@ -21,16 +21,20 @@ install_requires =
...
     matplotlib
     metpy
     numpy
     pandas
     panel
     param
+    pyproj
     pytest
-    regionmask
+    proj
```
Comment (bkg) [inline setup.cfg:36]: I think `pyproj` is used directly, but not `proj`.

**Review by bkg — COMMENTED** [review]
The `setup.cfg` should only include direct package dependencies, basically anything `import`ed. So, I don't think that `jinja2` or `proj` are needed there. I believe `jinja2` is present in those other lists to work around incompatible versions when it was implicitly installed, otherwise it wouldn't be needed there either.

**Comment by elehmer** [comment]
> The `setup.cfg` should only include direct package dependencies, basically anything `import`ed. So, I don't think that `jinja2` or `proj` are needed there. I believe `jinja2` is present in those other lists to work around incompatible versions when it was implicitly installed, otherwise it wouldn't be needed there either.

I see. Yes, I agree on both of these packages. I tried to remove `jinja2` from `environment.yml` and `requirements.txt` but panel errored out with import errors.

Ok, I'll remove these packages.

**Comment by elehmer** [comment]
Should we stick with version `0.0.1` in this file or increment it somehow?

**Comment by bkg** [comment]
That only gets bumped when a new package is officially tagged and released.

**Comment by elehmer** [comment]
> That only gets bumped when a new package is officially tagged and released.

Sounds good. Shall we merge this?

**Comment by bkg** [comment]
Yeah, go for it.

**Pull request merged**